Resources/doc/cookbook/recipe_knp_menu.rst

```diff
@@ -55,6 +55,31 @@ Add the controller route as an item of the menu:
                               route_params: { articleId: 3 }
                               label:        Article
 
+If you want to set access with different roles to your route you can configure this for each route. If current user is
+not granted for the specific route then it checks the global route for the group
```

Comment:
Do you mean route instead of roles?
A role is granted *on* a route (or any object) *to* somebody.